src/components/Docs/SnippetViewer/ListObjectsRequestViewer.tsx

```diff
@@ -0,0 +1,83 @@
+import { getFilteredAllowedLangs, SupportedLanguage, LanguageMappings } from './SupportedLanguage';
+import { defaultOperationsViewer } from './DefaultTabbedViewer';
+import assertNever from 'assert-never/index';
+import { TupleKey } from '@openfga/sdk';
+
+interface ListObjectsRequestViewerOpts {
+  user: string;
+  relation: string;
+  objectType: string;
+  contextualTuples?: TupleKey[];
+  skipSetup?: boolean;
+  allowedLanguages?: SupportedLanguage[];
+}
+
+function listObjectsRequestViewer(
+  lang: SupportedLanguage,
+  opts: ListObjectsRequestViewerOpts,
+  languageMappings: LanguageMappings,
+): string {
+  const { user, relation, objectType, contextualTuples } = opts;
+
+  switch (lang) {
+    case SupportedLanguage.PLAYGROUND:
```

Comment:
Can you remove playground from here?

If you want to keep it, just keep the note at the end that it is unsupported.